upup/pkg/fi/cloudup/gce/op.go

```diff
@@ -108,7 +108,8 @@ func waitForOp(op *compute.Operation, getOperation func(operationName string) (*
 		}
 		pollOp, err := getOperation(opName)
 		if err != nil {
-			klog.Warningf("GCE poll operation %s failed: pollOp: [%v] err: [%v] getErrorFromOp: [%v]", opName, pollOp, err, getErrorFromOp(pollOp))
+			klog.Warningf("GCE poll operation %s failed: pollOp: [%v] err: [%v]", opName, pollOp, err)
+			klog.Infof("getErrorFromOp: [%v]", getErrorFromOp(pollOp))
```

Comment:
```suggestion
			klog.Warningf("GCE poll getErrorFromOp: [%v]", getErrorFromOp(pollOp))
```

Reply:
This here is the problem

```
W1231 23:57:40.606721   75771 op.go:111] GCE poll operation operation-1704067055308-60dd707814e11-4ceef041-b4b17b9e failed: pollOp: [<nil>] err: [googleapi: Error 403: Quota exceeded for quota metric 'Heavy-weight read requests' and limit 'Heavy-weight read requests per minute per region' of service 'compute.googleapis.com' for consumer 'project_number:189277497252'.
Details:
[
  {
    "@type": "type.googleapis.com/google.rpc.ErrorInfo",
    "domain": "googleapis.com",
    "metadatas": {
      "consumer": "projects/189277497252",
      "quota_limit": "HeavyWeightReadRequestsPerMinutePerProjectPerRegion",
      "quota_limit_value": "1500",
      "quota_location": "us-east1",
      "quota_metric": "compute.googleapis.com/heavy_weight_read_requests_per_region",
      "service": "compute.googleapis.com"
    },
    "reason": "RATE_LIMIT_EXCEEDED"
  },
  {
    "@type": "type.googleapis.com/google.rpc.Help",
    "links": [
      {
        "description": "Request a higher quota limit.",
        "url": "https://cloud.google.com/docs/quota#requesting_higher_quota"
      }
    ]
  }
]
, rateLimitExceeded]
I1231 23:57:40.606831   75771 op.go:112] getErrorFromOp: [<nil>]
```

I need to hide that json block as it bloats the log file. It is a transient error that disappears after a minute usually for operations.

Reply:
OK